lib/puppet/type/sensu_silenced.rb

```diff
@@ -7,7 +7,10 @@
   desc <<-DESC
 @summary Manages Sensu silencing
 
-The name of `sensu_silenced` can be used to define `check` and `subscription`.
+The name of a `sensu_silenced` resource may not match the name returned by sensuctl.
+The name from sensuctl will take the form of `subscription:check`.
+If you wish to have a `sensu_silenced` resource name match sensuctl then define the name
+using the `subscription:check` format and do not define `subscription` or `check` properties.
```

Comment:
what happens if you use subscription:check and specify subscription or check parameters?